pkgs/desktops/mate/mate-tweak/default.nix

```diff
@@ -0,0 +1,82 @@
+{ stdenv
+, fetchFromGitHub
+, python3Packages
+, intltool
+, mate
+, libnotify
+, gtk3
+, gobject-introspection
+, wrapGAppsHook
+, glib
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "mate-tweak";
+  version = "20.04.0";
+
+  src = fetchFromGitHub {
+    owner = "ubuntu-mate";
+    repo = pname;
+    rev = version;
+    sha256 = "14cvh5hf61s0z75skrpz6r0z27jd01z100v2cqih2ck0pxv0483v";
+  };
+
+  nativeBuildInputs = [
+    wrapGAppsHook
+    intltool
+    python3Packages.distutils_extra
+    gobject-introspection
+  ];
+
+  buildInputs = [
+    gtk3
+    libnotify
+    glib
+    mate.mate-applets
+    mate.mate-panel
+    mate.marco
+    mate.libmatekbd
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    distro
+    pygobject3
+    psutil
+    setproctitle
+  ];
+
+  strictDeps = false;
+
+  dontWrapGApps = true;
+
+  postPatch = ''
+    # mate-tweak hardcodes absolute paths everywhere. Nuke from orbit.
+    find . -type f -exec sed -i \
+      -e s,/usr/lib/mate-tweak,$out/lib/mate-tweak,g \
+      {} +
+  '';
+
+  postInstall = ''
+    mv -v $out/${python3Packages.python.sitePackages}/nix/store/*/lib/* $out/lib
+    mv -v $out/${python3Packages.python.sitePackages}/nix/store/*/share/* $out/share
+    rm -r $out/${python3Packages.python.sitePackages}/nix
+  '';
```

Comment:
@jtojnar @FRidh Do either of you know how to work around or fix this packaging bug with distutils_extra?

Reply:
If the data_files are relative paths, they will be installed relative prefix, not site packages.

Reply:
they seem to have `{prefix}` https://github.com/ubuntu-mate/mate-tweak/blob/master/setup.py#L56

Reply:
I meant it must not be an absolute path. The `{prefix}/` part must be stripped to avoid the setuptools bug.

Reply:
I guess we can just do a sed operation. @luc65r can you replace the `{prefix}/` with nothing?

Reply:
(relevant issue: https://github.com/NixOS/nixpkgs/issues/23438)

Reply:
It does solve it!